src/dotnet/APIView/APIViewWeb/Pages/Assemblies/_CodeLine.cshtml

```diff
@@ -12,11 +12,16 @@
     };
 
     string foldableClass = Model.CodeLine.LineClass;
-    string [] foldableClassParts = Regex.Split(foldableClass, "\\s+");
-    bool isHeading = foldableClassParts.Any(x => x.EndsWith("-heading"));
-    bool isContent = foldableClassParts.Any(x => x.EndsWith("-content"));
-    string hiddenSectionClass = isContent ? "d-none" : "";
-    string sectionHeadingClass = isHeading ? "border-top" : "";
+    bool isHeading = false;
+    bool isContent = false;
+    string hiddenSectionClass = String.Empty;
+    if (!String.IsNullOrWhiteSpace(foldableClass))
+    {
+        string [] foldableClassParts = Regex.Split(foldableClass, "\\s+");
```

Comment:
Why does this need to be a regex split, verse a simple string.split with removeemptyentries? Generally string split will be more efficient than regex but I'm not sure how performance critical this code is.

Reply:
Ther is no particular reason am using Regex.Split. String.Split will still throw a null exception in this case though.

Reply:
Yes sure you still need to fix the issue you were trying to fix no matter which split option you use.